### PR TITLE
Aspose.Imaging Cloud v19.1

### PR DESCRIPTION
**Pull request by MaxCrank**

### CODE REVIEW

File: codegen/Templates/java/api.mustache [hunk 1]
```diff
@@ -169,10 +169,18 @@ public class {{classname}}
       {
           headerParams.Add("{{paramName}}", this.apiInvoker.toPathValue(request.{{baseName}})); // header parameter
       }{{/headerParams}}
-	  {{#hasBodyParam}}String postBody = SerializationHelper.serialize(request.{{bodyParam.baseName}}); // http body (model) parameter{{/hasBodyParam}}
+	  {{#hasBodyParam}}String postBody = null; ;
```
Comment (aifeigin) [inline codegen/Templates/java/api.mustache]: do we need two semicolons here ?

File: codegen/Templates/java/api.mustache [hunk 2]
```diff
@@ -169,10 +169,18 @@ public class {{classname}}
...
+      if (request.{{bodyParam.baseName}} instanceof String)
```
Comment (aifeigin) [inline codegen/Templates/java/api.mustache:173]: Could you please comment why do we need this change and give relevant example(s) of the response(s) ?

File: codegen/Templates/java/api.mustache (hunk 1 above)
Comment (MaxCrank) [reply codegen/Templates/java/api.mustache]: @aifeigin thanks, fixed.

File: codegen/Templates/php/api_test.mustache [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+     */
+    public function setUp()
+    {
+		$this->storage = new StorageApi();
+        $this->config = new Configuration();
+        $this->config->setHost("http://api-dev.aspose.cloud")->setAppSid("78b637f6-b4cc-41de-a619-d8bd9fc2b6b6")->setAppKey("3d588eb82b3d5a634ad3141f09b03629");
+        $this->storage->apiClient->apiKey = $this->config->getAppKey();
+        $this->storage->apiClient->appSid = $this->config->getAppSid();
+        $this->storage->apiClient->apiServer = $this->config->getHost();
+        $this->words = new WordsApi(null, $this->config);
```
Comment (aifeigin) [inline codegen/Templates/php/api_test.mustache]: Why do we use WordsApi here ?

Comment (MaxCrank) [reply codegen/Templates/php/api_test.mustache]: @aifeigin test templates aren't used, I'll remove them now, thanks. Additionally checked all SDK sources for mentioning the same, found couple of comments and fixed them.

File: codegen/Templates/java/api.mustache (hunk 2 above)
Comment (MaxCrank) [reply codegen/Templates/java/api.mustache:173]: @aifeigin this is made in scope of changing the default behavior from using post body to multipart form data. Post body isn't used starting from this version, and the suggested future use is sending the JSON data. If you check the generated API classes, you'll find that this part of the template is never used.

**Comment by aifeigin** [comment]
Please actualize Paid Support Helpdesk link to https://helpdesk.aspose.cloud/

**Comment by MaxCrank** [comment]
Fixed.

**Pull request merged**